practice/e-commerce-website/src/components/Button/Button.stories.tsx

```diff
@@ -11,8 +13,8 @@ type Story = StoryObj<typeof Button>;
 
 export const FirstStory: Story = {
```

Comment:
FirstStory not make sense to me, please update it all stories, example PrimaryButton or ...